pkgs/development/python-modules/testcontainers/default.nix

```diff
@@ -16,16 +17,27 @@
 
 buildPythonPackage rec {
   pname = "testcontainers";
-  version = "4.10.0";
+  version = "4.11.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "testcontainers";
     repo = "testcontainers-python";
     tag = "testcontainers-v${version}";
-    hash = "sha256-0Pd0GxG6Qh6qMJQSMRBaoE4dqFdWewNtdHf6te5vCeE=";
+    hash = "sha256-C3TWMf1u6RU9N7COL1DGgJQdvrpGnEWRgn9V0teyW/Q=";
   };
 
+  patches = [
+    # Bypass the socket address inference logic which fails in the nix sandbox:
+    # docker.errors.DockerException: Error while fetching server API version: ('Connection aborted.', FileNotFoundError(2, 'No such file or directory'))
+    # https://github.com/testcontainers/testcontainers-python/pull/832
+    (fetchpatch {
+      name = "fix-get-docker-socket";
+      url = "https://github.com/testcontainers/testcontainers-python/pull/832/commits/a6bcc12153c17b2aa498148af8b9a8258d86476b.patch";
```

Comment:
This URL returns 406 for me.

Reply:
They ended up yanking the release. Let me mark the PR as a draft.